kubernetes/cray-opa/templates/policies/spire.yaml

```diff
@@ -135,6 +135,13 @@ data:
           {"method": "POST", "path": `^/apis/hbtd/hmi/v1/heartbeat/.*$`},
           {{- end }}
           {"method": "GET", "path": `^/apis/hbtd/hmi/v1/params$`},
+        ],
+        "sbps-marshal": [
+        {{- if and (eq $.Values.opa.xnamePolicy.sbpsMarshal true) (eq $.Values.opa.xnamePolicy.enabled true) }}
+          {"method": "GET", "path": sprintf("^/apis/ims/v3/images/%v$", [parsed_spire_token.xname])},
```

Comment:
The IMS API doesn't have an xname as part of the API path here. Suggest simply taking the templated if out and using a static definition.